components/infobox/wikis/ageofempires/infobox_league_custom.lua

```diff
@@ -207,7 +206,6 @@ function CustomLeague:addToLpdb(lpdbData, args)
 
 	lpdbData.maps = Variables.varDefault('tournament_maps')
 
-	lpdbData.game = GameLookup.getName({args.game})
```

Comment:
this changes the stored value from name to identifier
might cause issues with consumers